tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -337,6 +337,7 @@ def find_and_replace_pattern(self, graph: Graph):
                 if input_type == 'FakeQuantize':
                     if fq['fq_config_priority'] == 'high' and input_node['fq_config_priority'] == 'low':
                         input_node['fq_group'] = fq['fq_group']
+                        input_node['fq_config_priority'] = 'high'
```

Comment:
Why should we change the `input_node['fq_config_priority']` from `low` to `high`?

Reply:
Since we override the fq_config of 'fq', we need to change the 'fq_config_priority' to that of 'fq' which is 'high' as the previous if condition said.

BTW, I modified the priority logic for GNA3 which is new.
Please newly review the "input_priority_types" option in tools/pot/openvino/tools/pot/configs/hardware/gna3.json.

In the previous change, FQ prioritize the fq config of 'outputs', so in the relu-conv case, the q-bit of activation for conv is q16 even we set to the 'performance' mode. (but the q-bit of weights of conv is still q8).
In my best understanding, GNA3 should unify the q-bit of activation and weights for conv.
So I add this "input_priority_types" option that gives higher priority to config of the activation and weights for conv.